pydicom/valuerep.py

```diff
@@ -466,20 +465,6 @@ class IS(int):
     Stores original integer string for exact rewriting of the string
     originally read or stored.
     """
-    if compat.in_py2:
-        __slots__ = ['original_string']
-
-        # Unlikely that str(int) will not be the
-        # same as the original, but could happen
-        # with leading zeros.
-
-        def __getstate__(self):
-            return dict((slot, getattr(self, slot)) for slot in self.__slots__
-                        if hasattr(self, slot))
-
-        def __setstate__(self, state):
-            for slot, value in state.items():
-                setattr(self, slot, value)
 
     def __new__(cls, val):
```

Comment:
I think we can get rid of `PersonName`, PersonNameUnicode` and `PersonNameBase` (maybe keeping an alias to `PersonName3` for compatibility).

Reply:
I've been keeping PersonName3 so far.  I'm not sure what user code might have referred to in the past, likely none of these, except perhaps in `isinstance` checks.  Because of that, I thought maybe renaming PersonName3 to PersonName might be risky.  But I'm open to ideas on what to call everything.

Reply:
I would rename `PersonName3` to `PersonName` and change all references in code and documentation, and add a `PersonName3` alias for backwards compatibilty. With Python 2 support gone, `PersonName3` doesn't make much sense anymore. There has been a `PersonName` alias in most places in the code anyway.